todo_app/app.py

```diff
@@ -15,8 +17,18 @@ def __init__(self, id):
 def create_app():
         app = Flask(__name__)
         app.config['LOGIN_DISABLED'] = os.getenv('LOGIN_DISABLED') == 'True'
+        app.config['LOG_LEVEL'] = os.getenv('LOG_LEVEL')
```

Comment:
Minor, but it might make sense to specify a default value for this here (as below) so that we can choose a sensible option if the user doesn't specify one:
```suggestion
        app.config['LOG_LEVEL'] = os.getenv('LOG_LEVEL', 'INFO')
```